acceptance/testdata/pack_current/inspect_builder_output.txt

```diff
@@ -1,4 +1,5 @@
 Inspecting builder: '{{.builder_name}}'
+
```

Comment:
Curious about this because I would have expected tests to fail before this change (unless I haven't looked far enough ahead in this PR for code updates).

Reply:
Nvm, see the corresponding change.